src/Cms/App.php

```diff
@@ -836,6 +847,16 @@ public function languages(bool $clone = true)
         return $this->languages = Languages::load();
     }
 
+    /**
+     * Load a part of Kirby
```

Comment:
Seems to be a bit misleadingly phrased. Maybe more `Access Kirby's part loader`?